docs/source/gcp/examples/appengine-domain-mapping.rst

```diff
@@ -0,0 +1,24 @@
+App Engine - Audit domain mappings being added
+===============================================
+Custodian can audit new mappings being added. Note that the ``notify`` action requires a Pub/Sub topic to be configured.
+
+In the example below, the policy notifies users if the ``CreateDomainMapping`` action appears in the logs.
+
+.. code-block:: yaml
+
+    policies:
+      - name: gcp-app-engine-domain-mapping-audit-new-mappings
+        resource: gcp.app-engine-domain-mapping
+        mode:
+          type: gcp-audit
+          methods:
+            - "google.appengine.v1beta.DomainMappings.CreateDomainMapping"
+        actions:
```

Comment:
this example should be removed, its generic. i think longer term (aka in future) domain mapping should be a filter to allow for this to be used usefully atm its just conveys a set of ids afaict.

Reply:
Done.